src/ResourceManager/DevSpaces/Commands.DevSpaces/help/AzureRM.DevSpaces.md

```diff
@@ -0,0 +1,25 @@
+---
+Module Name: AzureRM.DevSpaces
+Module Guid: 
```

Comment:
Please fill out these fields - use other AzureRM.<Module>.md files for examples. GUID can be found in the csproj file under projectguid.